src/SignTool/SignTool/SignTool.RealSignTool.cs

```diff
@@ -79,7 +79,7 @@ public override void RemovePublicSign(string assemblyPath)
                     }
 
                     stream.Position = peReader.PEHeaders.CorHeaderStartOffset + OffsetFromStartOfCorHeaderToFlags;
-                    writer.Write((UInt32)(peReader.PEHeaders.CorHeader.Flags | CorFlags.StrongNameSigned));
+                    writer.Write((UInt32)(peReader.PEHeaders.CorHeader.Flags & ~CorFlags.StrongNameSigned));
```

Comment:
>& ~ [](start = 77, length = 3)

Was this always wrong?

Reply:
Yes. @nguerrera pointed this out to me a while ago.